tests/base.py

```diff
@@ -236,13 +233,18 @@ def expected_replication_keys(self):
                 for table, properties
                 in self.expected_metadata().items() if table in incremental_streams}
 
+    def expected_start_date_stream_to_key(self):
+        return {table: properties.get(self.START_DATE_KEY)
+                for table, properties in self.expected_metadata().items()
+                if properties.get(self.START_DATE_KEY)}
+
     def expected_automatic_fields(self):
         auto_fields = {}
         for k, v in self.expected_metadata().items():
             auto_fields[k] = v.get(self.PRIMARY_KEYS, set()) | v.get(self.REPLICATION_KEYS, set())
         return auto_fields
 
-    def select_all_streams_and_fields(self, conn_id, catalogs, select_all_fields: bool = True, exclude_streams=[]):
+    def select_all_streams_and_fields(self, conn_id, catalogs, select_all_fields: bool = True, exclude_streams=None):
```

Comment:
Unless this is always passed in, it'll need a fallback `exclude_streams = exclude_streams or []` or something similar